branch_name_verification_hook.py

```diff
@@ -5,8 +5,12 @@
 
 excluded_branches = {'master', 'develop', 'staging', 'test'}
 
-output, error = run_shell('git', 'symbolic-ref --short HEAD', False)
-branch_name = output.decode('utf8')[:-1]
+wiki_url = 'https://github.com/space-concordia-robotics/robotics-prototype/wiki/Git-Workflow-and-Conventions'
+error_msg = 'Branch name is not named properly, please see wiki for formating: ' + wiki_url
```

Comment:
I prefered having the error message inside that function since its very odd that you need to pass an error message as an argument to a function that its role is to check if a branch is good or not. Also, it seems like you don't use it anywhere else.

Reply:
I did this in order to avoid recreating the `wiki_url` and `error_msg` variables in the commit message hook. So I did actually reuse it in the commit message hook. I guess it's not that big of a deal tho, so I'll just change that.